sdk/storage/azure_storage_blob/Cargo.toml

```diff
@@ -11,6 +11,10 @@ rust-version.workspace = true
 
 [dependencies]
 azure_storage_common.workspace = true
+azure_core = { workspace = true, features = ["xml"] }
+serde = { workspace = true }
+time = { workspace = true }
+typespec_client_core = { workspace = true, features = ["derive"] }
```

Comment:
@heaths long-term do we expect/want Azure SDKs to have a direct dependency on this? There are a few things in `typespec_client_core` that haven't been re-exported from `azure_core` (e.g. the `create_enum` macros).

Reply:
I don't see any harm, but I don't have a strong opinion either way. @RickWinter @ronniegeraghty @JeffreyRichter what do you think? Should we necessarily re-export everything in `typespec_client_core` from `azure_core`? Hmm, having just typed that I'm actually thinking we shouldn't. It's an unnecessary burden. I only did it originally for migration and wanted to get rid of them (mostly) eventually. Since we generate and expect very little hand-coding, it wouldn't be hard to rename those crates later should we need to.

I wouldn't hold this PR up for that, though. Something we should discuss further offline.